src/z3c/pt/README.rst

```diff
@@ -390,7 +390,7 @@ Adapters build upon).
   ...     def parent(self):
   ...         return self.context.parent
 
-  >>> function_namespaces.registerFunctionNamespace('ns1', ns1)
+  >>> function_namespaces.namespaces['ns1'] = ns1
```

Comment:
This makes me a bit sad, but I've no energy to even finish this comment.

Reply:
I didn't mean to make anyone sad 😢 

This is because the AdapterNamespace from `zope.pagetemplate` doesn't expose any methods like the version in this package does, but they both have the `namespaces` attribute. This lets us be compatible with both versions.